semver.md

```diff
@@ -102,8 +102,10 @@ intended compatibility requirements as denoted by its associated
 normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7,
 1.0.0-x.7.z.92.
 
-1. Build metadata MAY be denoted by appending a plus sign and a series of dot
```

Comment:
Excellent point. To be honest, I wasn't interested in pre-releases, so I skipped that section when reading the spec. It makes sense to keep them consistent, so I'll update my PR.